static/sass/_layout-cube.scss

```diff
@@ -0,0 +1,9 @@
+@media (min-width: $breakpoint-medium) {
+  .large-column {
```

Comment:
I don't think these classes will scale or be reusable in other tables around ubuntu.com. Could you make the table column with bespoke to the cube table. Similar to this: https://github.com/canonical-web-and-design/maas-ui/blob/master/legacy/src/scss/tables/_patterns_table-controllers.scss

Reply:
Ok sure will do, thanks